locales/en/yourloops.json

```
@@ -416,7 +419,8 @@
   "time-in-range-cgm-one-day": "**Time In Range:** Time spent in range, based on CGM readings.",
   "time-out-of-range": "Time spent out of range",
   "time-out-of-range-target": "Time spent out of the target range",
-  "time-out-of-range-target-tooltip": "By default, this alert threshold is set to:\n>25% time spent away from target over the given period.\n\nThis value can be modified either at the level of all remotely monitored patients, or patient by patient.",
+  "time-out-of-range-target-tooltip1": "Out of the range: {{percentage}}%.",
+  "time-out-of-range-target-tooltip2": "Alert triggered when more than {{threshold}}% of time over the period considered are off target ({{min}}-{{max}}{{unit}}).",
```

Need a space between a number and its unit
```suggestion
  "time-out-of-range-target-tooltip2": "Alert triggered when more than {{threshold}}% of time over the period considered are off target ({{min}}-{{max}} {{unit}}).",
```